eventhandler.py

```diff
@@ -11,9 +11,6 @@
     'closed': 'on_pr_closed',
     'labeled': 'on_issue_labeled'
 }
-_test_path_roots = ['a/', 'b/']
-
-DIFF_HEADER_LINE_START = 'diff --git '
```

Comment:
These methods ended up in the _APIProvider_ class (no longer part of _EventHandler_)... better cohesion, because they abstract the info provided in the API's diffs. Is that what you were getting at? Or, is there a different approach you're suggesting? Thanks!